CONTRIBUTING.md

```diff
@@ -93,11 +93,12 @@ The following project structure should be followed:
 - Run the project locally, make sure the server is listening to requests
 - Navigate to `https://github.com/login/oauth/authorize?client_id=<GITHUB_CLIENT_ID>`
 - Authorize the application
-- Copy the response cookie named rds-session from the redirected request <HOST>/auth/github/callback?code=<SOME_AUTOGENERATED_CODE>
+- Once authorized, check browser's cookies section. There's your 🍪 named `rds-session` (`rds-session-development` in development mode).
```

Comment:
> Once authorized, check browser's cookies section. Copy the value of the cookie named `rds-session` (`rds-session-development` for development mode).

This fits well with the grammar, language and style of existing documentation. What do you think?

Reply:
Sounds good to me.